frontend/src/metabase/dashboard/components/add-card-sidebar/QuestionPicker.jsx

```diff
@@ -28,17 +30,13 @@ QuestionPicker.propTypes = {
   onSelect: PropTypes.func.isRequired,
   collectionsById: PropTypes.object,
   getCollectionIcon: PropTypes.func,
-  initialCollection: PropTypes.number,
 };
 
-function QuestionPicker({
-  onSelect,
-  collectionsById,
-  getCollectionIcon,
-  initialCollection,
-}) {
+function QuestionPicker({ onSelect, collectionsById, getCollectionIcon }) {
+  const dashboard = useSelector(getDashboard);
```

Comment:
Not sure if it'd be better to do these instead, but I keep it simple for now.
1. `useSelector(state => getDashboard(state).collection)`
2. or `useSelector(getDashboardCollection)` (a new selector)